lib/ht/search_client/remote.rb

```diff
@@ -11,7 +11,7 @@ class Remote
       :order, :per_page, :types, :features,
       :bedrooms, :page, :guests, :price_min,
       :price_max, :currency, :aggregations, :range_aggregations,
-      :excluding_ids
+      :excluding_ids, :ib, :ib_ee
```

Comment:
They don't need to be. We decided to stick use abbreviations for the AB test. If we decide to productionize we'll use more human friendly language.